apps/studio/pages/integrations/vercel/install.tsx

```diff
@@ -227,11 +231,23 @@ const VercelIntegration: NextPageWithLayout = () => {
                 </AlertDescription_Shadcn_>
               </Alert_Shadcn_>
             )}
+            {hasVercelManagedOrgSelected && (
+              <div className="prose text-sm text-red-900">
```

Comment:
is the div required? can just apply the class names to <p>

`<p className="prose text-sm text-red-900">`

Reply:
Done